team-config.yml

```diff
@@ -705,3 +705,11 @@ ts:
     build_notices_channel: "#ccd-master-builds"
   tags:
     application: ts-translation-services
+sptribs:
+  team: "Special Tribunals"
+  namespace: "sptribs"
+  slack:
+    contact_channel: "#special-tribunals-dev-channel"
+    build_notices_channel: "#special-tribunals-builds"
+  tags:
+    application: sptribs-case-api
```

Comment:
can you confirm this with "Bruce Rawstorne" on slack please?

Reply:
Have asked him to review the PR, awaiting a response